tests/README.md

```diff
@@ -0,0 +1,38 @@
+## Test Organization
+This folder contains all the tests for the Brevitas repository.
+
+Everytime a new function is added or modified, it is highly recommeded to adapt the associated test cases if present, or to add new ones if tests are missing.
+
+We are in the process of expanding and re-organising the test suite, thus information and descriptions contained here might change in the future.
+
+Brevitas test suite is built around the following packages:
+- Pytest
+- Hypothesis
+- Pytest Cases
+- Mock (Pytest Mock)
+
+Please feel free to familiriasize with their basic usage before contributing to new tests on Brevitas.
+
+### Folder Structure
```

Comment:
This is correct, but we should also add a suffix _tests to each folder in a separate PR cause right now it can cause import conflicts otherwise.

Reply:
I will open an issue about this